packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.tsx

```diff
@@ -171,6 +186,25 @@ export class GeovEntityPropsByPredicate {
     }
   }
 
+  componentDidLoad() {
+    // Display Expand button if need. The page must be loaded in order to have the measurements
+    const shadowRoot = this.hostElement.shadowRoot;
+    if (shadowRoot) {
+      const itemLiteral = shadowRoot.querySelector('.literal-container .literal-collapsed');
+      const itemButton = shadowRoot.querySelector('.literal-container .item-button') as HTMLElement;
+
+      if (itemLiteral && itemButton) {
+        const labelLiteral = itemLiteral.querySelector('ion-label');
+        if (labelLiteral) {
+          // If size of text > size of container
+          if (labelLiteral.scrollWidth > labelLiteral.clientWidth) {
+            itemButton.style.display = 'block';
+          }
+        }
+      }
+    }
+  }
```

Comment:
Great work! 

Nevertheless I see disadvantages: When the page size is changed, this script does not rerun.

I could also imagine to simplify this: Just add a ["open" button](https://ionicframework.com/docs/api/item#buttons-in-items) to the following parts to open the string in a modal, in the case of xsd strings and lang strings:

```javascript
 private renderXsdStringLiteral(item: Bindings) {
    return <ion-label>{item.entity.value}</ion-label>;
  }
  private renderLangStringLiteral(item: Bindings) {
    return (
      <ion-label>
        <h2>{item.entity.value}</h2>
        <p>@{item.entity?.['xml:lang']}</p>
      </ion-label>
    );
  }
```

This would also resolve the difficulties with overriding the ion-item style.

It will also be more stable for very long strings, since the modal can be broader then the narrow screen space it currently has.

@perrauda and @flicksolutions What do you think about radically changing the approach?

Reply:
I agree, that a modal would be more beautiful and robust. I'd also argue, that we should only display the open button if there is overflowing text. one could add a eventlistener to the resize event and call the calculation for the overflow on resize: https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event

Reply:
I modified this part so that I could reuse the calculation, but I didn't manage to associate it with the 'resize' event (line 184), which is probably a misunderstanding on my part.
Otherwise, the button still only appears if the text is overflow, but it brings up the modal instead of extending the ion-card.
What remains to be done is to link it to the resize event, perhaps improve the ion-modal (too much white space).